Homework12/Homework12/WeatherResponse.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework12
+{
+    public class WeatherResponse
+    {
+        public Location location { get; set; }
```

Comment:
Super, vidim, ze aby se ti to spravne deserializovalo, tak sis poradil tak, ze jsi pojemnoval properties, aby odpovidali nazvum v ziskanem jsonu.

Naming convence muze byt jakkakoliv u externich dat a idealne by jsme se u sebe v projektu chteli drzet te klasicke C#pove (tedy tady by bylo `Location` a ne `location`).

Proc by ti to ale samo od sebe nefungovalo, kdyz bys pouzil `Location`?
Jde o to, ze se pri deserializaci nedokaze urcit, ke ktere property se ma hodnota priradit pokud to nematchuje 1:1

Protoze vetsinou to nematchuje 1:1, tak se pomoci atributu doplnuje informace o tom, jak se to ma (at uz pro deserializaci, nebo serializaci) jmenovat.

Takto by to vypadalo napsane spravne, aby deserializer vedel, jak to spravne namapovat:
```suggestion
        [JsonPropertyName("location")]
        public Location Location { get; set; }
```

Reply:
A tady nejaky clanek k tem atributum, pomoci kterych reknes (de)serializeru, jak namapovat:
en: https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties
cz: https://learn.microsoft.com/cs-cz/dotnet/standard/serialization/system-text-json/customize-properties